homeassistant/components/aten_pe/switch.py

```diff
@@ -0,0 +1,114 @@
+"""The ATEN PE switch component."""
+
+import logging
+
+from atenpdu import AtenPE
+import voluptuous as vol
+
+from homeassistant.components.snmp.const import (
```

Comment:
Don't import constants from other integrations.
Define it in your own integration.